core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -5709,6 +5709,19 @@ public boolean isBangEqualAllowed() {
         .withConformance(lenient).ok();
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5486">[CALCITE-5486]
+   * SubQuery not support HAVING alias in where condition</a>. */
+  @Test void testHavingAliasInCondition() {
```

Comment:
Should we also add a test in `SqlToRelConverterTest` to ensure we get the right plan?

Reply:
Yes, done.